### PR TITLE
Add config for running tests and gem build in solano

### PR DESCRIPTION
**Pull request by glittershark**
@jsjohnst

### CODE REVIEW

File: Gemfile.rails42 [hunk 1]
```diff
@@ -2,3 +2,4 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'rails', '~> 4.2.0'
```
Comment (jsjohnst) [inline Gemfile.rails42]: why is this file here?

Comment (glittershark) [inline Gemfile.rails42]: Comes from turbolinks, because they test with all versions of Rails. I'll kill it for us, though

**Comment by jsjohnst** [comment]
:shipit:
